images/windows/Windows2022-Readme.md

```
@@ -205,350 +212,342 @@ Note: MSYS2 is pre-installed on image but not added to PATH.
+| Nginx  | 1.27.4  | C:\tools\nginx-1.27.4\conf\nginx.conf | nginx       | Stopped       | 80         |
 
 ### Visual Studio Enterprise 2022
-| Name                          | Version       | Path                                                     |
-| ----------------------------- | ------------- | -------------------------------------------------------- |
-| Visual Studio Enterprise 2022 | 17.9.34902.65 | C:\Program Files\Microsoft Visual Studio\2022\Enterprise |
+| Name                          | Version         | Path                                                     |
+| ----------------------------- | --------------- | -------------------------------------------------------- |
+| Visual Studio Enterprise 2022 | 17.13.35825.156 | C:\Program Files\Microsoft Visual Studio\2022\Enterprise |
 
 #### Workloads, components and extensions
 | Package                                                                   | Version         |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines
null

(MD058, blanks-around-tables)